editor/plugins/path_2d_editor_plugin.h

```diff
@@ -91,7 +95,10 @@ class Path2DEditor : public HBoxContainer {
 	void _handle_option_pressed(int p_option);
 
 	void _node_visibility_changed();
-	friend class Path2DEditorPlugin;
+
+	void confirm_clear_points();
+	void clear_curve_points(Path2D *p_path2d);
+	void restore_curve_points(Path2D *p_path2d, const PackedVector2Array &p_points);
```

Comment:
```suggestion
	void _confirm_clear_points();
	void _clear_curve_points(Path2D *p_path2d);
	void _restore_curve_points(Path2D *p_path2d, const PackedVector2Array &p_points);
```
Private methods should begin with underscore.